app/uk/gov/hmrc/hec/repos/HECTaxCheckStore.scala

```diff
@@ -54,7 +54,11 @@ trait HECTaxCheckStore {
 
   def deleteAll()(implicit hc: HeaderCarrier): EitherT[Future, Error, Unit]
 
-  def getTaxCheckCodeByKey[A](key: String, value: A)(implicit
+  def getAllTaxCheckCodesByExtractedStatus(status: Boolean)(implicit
+    hc: HeaderCarrier
+  ): EitherT[Future, Error, List[HECTaxCheck]]
+
+  def getAllTaxCheckCodesByCorrelationId(correlationId: String)(implicit
```

Comment:
`getAllTaxCheckCodesByFileCorrelationId` would be clearer here I think

Reply:
updated [here](https://github.com/hmrc/hec/pull/42/commits/7e13dddc516c16bc282fb4f4c246c82bf3e80885)